### PR TITLE
Create end2end_tfkeras_serialize.py

### PR DESCRIPTION
**Pull request by microprediction**
When I read the Keras example I was initially unsure how to avoid the use of subprocess/saving the keras model to file. This is a minor modification to the existing example as I suspect some users would prefer not to go via disk, or may not have entitlements to do so.

### CODE REVIEW

File: examples/end2end_tfkeras_serialize.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+"""
+This example builds a simple model without training.
+It is converted into ONNX directly, by serializing to byte string.
+Predictions are compared to the predictions from tensorflow to check there is no
+discrepencies. Inferencing time is also compared between
+*onnxruntime*, *tensorflow* and *tensorflow.lite*.
+"""
+
+# from https://github.com/microprediction/tensorflow-onnx/blob/master/examples/end2end_tfkeras.py
```
Comment (TomWildenhain-Microsoft) [inline examples/end2end_tfkeras_serialize.py:11]: I don't think we want this reference to an external repo.

**Review by TomWildenhain-Microsoft — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution. I agree that having an example using `from_keras` would be great. I'd favor renaming the existing tutorial to be called `end2end_tf_saved_model` and edit `end2end_tf_keras` accordingly.

**Comment by microprediction** [comment]
A renaming & cut and pate error, but hopefully the end result is okay. 
I added a few lines for the M1 folks.

File: examples/end2end_tfkeras_serialize.py (hunk 1 above)
Comment (microprediction) [reply examples/end2end_tfkeras_serialize.py]: shall delete

**Comment by guschmue** [comment]
inactive - closing

**Pull request closed without merging**